setup.py

```diff
@@ -4,17 +4,21 @@
 """The setup script."""
 
 from setuptools import find_packages, setup
+from typing import Dict, List
 
 with open("README.md") as readme_file:
     readme = readme_file.read()
 
-format_libs = {
-    "base-imageio": "imageio[ffmpeg]~=2.9.0",
-    "lif": "readlif~=0.6.1",
-    "czi": "aicspylibczi~=3.0.2",
+format_libs: Dict[str, List[str]] = {
+    "base-imageio": ["imageio[ffmpeg]~=2.9.0", "Pillow~=8.2.0,!=8.3.0"],
+    "lif": ["readlif~=0.6.1"],
+    "czi": ["aicspylibczi~=3.0.2"],
 }
 
-all_formats = [v for v in format_libs.values()]
+all_formats: List[str] = []
```

Comment:
You might be able to just do `all_formats = itertools.chain(format_libs.values())` !

Reply:
I knew there was a shorthand! I'll blame it being early in the morning for me... But for now I am going to keep as is.

Reply:
I would say you could also use `all_formats.extend(deps)` at the second level. But yea it's good 👍🏼